HopsanCore/src/Quantities.cpp

```diff
@@ -56,13 +55,18 @@ hopsan::QuantityRegister::QuantityRegister()
 
     registerQuantity("Density", "kg/m^3");
 
-    registerQuantity("Frequency", "rad/s");
+    registerQuantity("Frequency", "Hz");
+    registerQuantity("AngularVelocity", "rad/s");
     registerQuantity("Time", "s");
 
     registerQuantity("Temperature", "K");
 
+    registerQuantity("Resistance", "Ohm");
```

Comment:
Why Ohm with a capital O? I don know myself, but some quick googling and Wikipediaing shows me that it seem to be written with lower-case ohm

Reply:
Haha, yes I made the same Googling. The reason is cosmetic; units mOhm, kOhm, MOhm etc becomes easier to read than mohm, kohm and Mohm. Besides, I think it's not recommended with short prefix at all. Instead milliohm and kiloohm should be used. Personlly, I was just fine with mOhm, kOhm etc. Easy to read and understood by most.